kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/Task.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.patterns.factory.tasks;
+
+public interface Task {
+
+    void executeTask();
+    String getTaskName();
+    boolean isTaskExecuted();
```

Comment:
Bardziej pasowaloby aby task byl klasa abstrakcyjna, gdzie wrzucilbys powtarzalne pola i metody z klas dziedziczacych. 
*Ale widzialem ze to wymog z materialow Kodilli